misk/src/test/kotlin/misk/metrics/backends/stackdriver/StackDriverReporterTest.kt

```diff
@@ -55,8 +57,11 @@ internal class StackDriverReporterTest {
         }
     }
 
-    @get:Rule
-    val miskTestRule = MiskTestRule(MetricsModule(), FakeClockModule(), TestModule())
+    @Modules val modules = ModuleProvider(
```

Comment:
Since modules themselves are composable I wonder if its easier to just do this:

```
@ActionTestModule val actionTestModule = Modules.combine(
        MetricsModule(),
        FakeClockModule(),
        TestModule()
);
```

Reply:
Also something I'll look into in another PR. And yes, good point.

Reply:
Done in this PR since I had to rebase anyway.